ci/kind/test-e2e-kind.sh

```diff
@@ -53,6 +53,8 @@ proxy=true
 endpointslice=false
 np=true
 coverage=false
+longcasesonly=false
```

Comment:
Could you add `longcaseonly` and `modeirrelevant` into `Usage` of the script? Also, some explanations for these 2 will be helpful to other developers.
Does `modeirrelevant` mean `antctl,bandwidth,basic,batch,security,supportbuddle,tls`? I am not sure if it is good name and if I understand it correctly. Maybe `minorfunctionsonly`?

Reply:
done, I refined the script to use existing mode parameter "--encap-mode" and new "--time-consuming-cases-only" option to generate skip list now and add an description in `Usage`